### ISSUE TITLE
Hold the icon position at the end of IconTextView

### ISSUE THREAD

**Issue by brunomb** [guilhermesgb/ShiftableEditText#4]
Would be nice the option to hold the icon position when the text in a IconTextView need scrolling.

**Comment by xibaca** [comment]
I agree! It would be nice to be able to control the icon behavior via a new attr `iconBehavior`. I can only think of two behaviors as of now: `marquee` and `still`, but this thread shall be open for new suggestions!
